src/main/java/com/softwire/todos/JiraClient.java

```diff
@@ -105,27 +97,6 @@ public void updateComment(Issue issue, Comment comment) throws Exception {
         }
     }
 
-    /**
-     * This wrapper just exposes somes protected methods
-     */
-    private class ClientWrapper extends AbstractAsynchronousRestClient {
-        public ClientWrapper(AbstractAsynchronousRestClient inner) throws Exception {
-            super(getApacheClient(inner));
-        }
-
-        public <T> Promise<Void> put2(final URI uri, final T entity, final JsonGenerator<T> jsonGenerator) {
-            return super.put(uri, entity, jsonGenerator);
-        }
-
-        public final Promise<Void> delete2(final URI uri) {
-            return delete(uri);
-        }
-
-        public <T> Promise<T> getAndParse2(final URI uri, final JsonParser<?, T> parser) {
-            return getAndParse(uri, parser);
-        }
-    }

```

Comment:
Thanks
This was obsoleted by 21b73e571d / e0b14f354a8

See https://github.com/RichardBradley/jira-rest-java-client/tree/rtb-fork